cypress/integration/logout.spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Console logout', () => {
+  it('succeeds when logged in properly', () => {
+    cy.login()
+    cy.visit('/console')
+    cy.get('header').within(() => {
+      cy.findByText('admin').click()
```

Comment:
For me it reads like: there is the `profile-dropdown` entry in the `header` and it should display the users id
```suggestion
      cy.findByTestId('profile-dropdown')
      	.should('contain', 'admin')
      	.click()
```
this should possible as `should` returns the same subject it was given